src/libraries/System.IO.FileSystem/tests/Enumeration/AttributeTests.cs

```diff
@@ -123,6 +123,7 @@ public void IsHiddenAttribute()
 
             fileOne.Create().Dispose();
             fileTwo.Create().Dispose();
+
             if (PlatformDetection.IsWindows)
                 fileTwo.Attributes = fileTwo.Attributes | FileAttributes.Hidden;
```

Comment:
Shouldn't this also be performed on other platforms?

Reply:
No - On Linux you hide a file by prepending a dot to the filename (see line 122 in this file). On Windows, you need to set the flag.

Reply:
You pointed out the documentation says it is supported in Linux, Windows and MacOS. Let me degug this unit test in WIndows and Unix to see if it's truly supported there, and if so, then the if should be removed, and maybe a separate test should be added for Linux and MacOS (because of the "." prefix in the name, which already hides the file).

Reply:
So in Linux, if I remove the code that prepends a "." to the filename, and attempt to set FileAttributes.Hidden to fileTwo, the test fails, because
`Interop.Sys.CanSetHiddenFlag` returns false.
The documentation needs to be updated for Linux.
I'll test MacOS and Windows now.

Reply:
It works on OSX. I actually found a test in FileSystemEntry which would prevent the user from collecting an enumeration of hidden files. I fixed it now.
Next: Windows.

Reply:
Works on Windows too.